docs/best-practices.md

```diff
@@ -132,3 +132,16 @@ the package if they want to experience as smooth upgrade as possible.
 If you're writing an actor for the RHEL 7 to RHEL 8 in-place upgrade workflow and you really need to have some package
 installed, then the only acceptable packages to depend on are the the ones available in the minimal installation of
 RHEL 7 (packages from the @core group + their dependencies).
+
+## When something goes wrong stop the execution of your actor in the right way
```

Comment:
"Use convenience exceptions to stop the actor's execution"

I think the "when" part doesn't necessarily be in the headline. That can be found in the details below.

Reply:
+1